src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -28,6 +28,7 @@ module UpdateAlternatives
   class MainDialog < UI::Dialog
     def initialize
       @alternatives_list = UpdateAlternatives::Alternative.all.reject(&:empty?)
+      @one_choice_filter = true
```

Comment:
to be honest for me one_choice_filter is a bit confusing name as it indicate you filter only one choice...maybe better name is `multi_choice_only` variable?